File: kody/hello.cpp
/*
 * hello.cpp
 * 
 * Copyright 2017  <>
 */


#include <iostream>

using namespace std;

#define ROK_TERAZ 2017
#define ROK_CPP 1983

void parzyste(int a){
    int c=0;
    cout<<"Liczby parzyste: "<<endl;
    for(int i=0; i<=a; i=i+2){
        cout<<i<<" ";
        c++;
    }
    cout<<"Ilość parzystych: "<<c;
}


int main(int argc, char **argv)
{   
    string imie;
    int wiek,rok,liczba;
	cout<<"Imię?"<<endl;
    cin>>imie;
    cout<<"Witaj "<<imie<<endl<<"Ile masz lat?"<<endl;
    cin>>wiek;
    rok = ROK_TERAZ-wiek;
    cout<<"Urodziłeś się w "<<rok<<" roku.";
    if (rok>ROK_CPP){
        cout<<"Jestem starszy!"<<endl;
        }
    else if (rok<ROK_CPP){
        cout <<"Jestem młodszy"<<endl;
        }
    else {
        cout << "Mamy tyle samo lat!"<<endl;
        }
    cout<<"Z jakiego zakresu wypisać liczby parzyste? "<<endl;
    cin>>liczba;
    parzyste(liczba);
    
	return 0;
}

File: kody/hello.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-
#
#  hello.py
#  
#  Copyright 2017  <>
#  
ROK_TERAZ = 2017
ROK_PY = 1991

def parzyste(n):
    print("Liczby parzyste: ")
    ile = list(range(0, n+1, 2))
    print(ile,"Ilość parzystych: ")
    return len(ile)

def main(args):
    
    imie = input("Imię? ")    
    print("Witaj",imie)
    wiek = int(input("Ile masz lat? "))
    rok=ROK_TERAZ-wiek
    print("Urodziłeś się w ",rok," roku.")
    if (rok>ROK_PY):
        print("Jestem starszy!")
    elif (rok<ROK_PY):
        print("Jestem młodszy!")
    elif (rok==ROK_PY):
        print("Mamy tyle samo lat!")
    n = int(input("Z jakiego zakresu wypisać liczby parzyste? "))
    print(parzyste(n))
    return 0

if __name__ == '__main__':
    import sys
    sys.exit(main(sys.argv))
